src/components/box/Box.jsx

```diff
@@ -44,6 +50,11 @@ const Box = ({color, padding, full, children, border = !color, imgSrc, noMinHeig
 
   return (
     <div {...props} className={boxClass}>
+      {onClose ?
+        <div className="sg-box--message__close" onClose={onClose}>
```

Comment:
`sg-box--message__close` is a wrong bem selector :) I think `sg-box__close` is good :)

Reply:
i agree 👍